docs/source/index.rst

```diff
@@ -75,6 +77,12 @@ or install it using your system package manager, e.g.::
 
     sudo apt-get install python3-pyqt5
 
+or::
+
+    sudo apt-get install python-pyqt
+    sudo apt-get install python3-pyqt
+
+if PyQt4 is desired.
```

Comment:
@takluyver Thanks for the correction. I don't feel strongly about keeping in or taking out. Since the code base still mentions Qt 4, it might be good to just leave it in but corrected.